atlasdb-perf/src/main/java/com/palantir/atlasdb/performance/backend/DatabasesContainer.java

```diff
@@ -23,15 +23,15 @@
 
 public final class DatabasesContainer implements AutoCloseable {
 
-    public static DatabasesContainer startup(List<KeyValueServiceType> backends) {
+    public static DatabasesContainer startup(List<KeyValueServiceTypeInterface> backends) {
         List<DockerizedDatabase> dbs = Lists.newArrayList();
         try {
-            for (KeyValueServiceType backend : backends) {
+            for (KeyValueServiceTypeInterface backend : backends) {
                 DockerizedDatabase db = DockerizedDatabase.start(backend);
                 Awaitility.await()
-                        .atMost(Duration.ONE_MINUTE)
-                        .pollInterval(Duration.ONE_SECOND)
-                        .until(() -> KeyValueServiceType.canConnect(backend, db.getUri().getAddress()));
+                        .atMost(Duration.FIVE_MINUTES)
+                        .pollInterval(Duration.ONE_MINUTE)
```

Comment:
ONE_MINUTE was too short for Oracle, and the five minute wait (which happens only if the connection fails) is short compared to a successful run of the benchmarks.